docs/how-to/deploy_the_sdcore_user_plane.md

```diff
@@ -1,15 +1,14 @@
-# Deploy the SD-CORE UPF
+# Deploy the SD-CORE User Plane
 
-This guide explains how to deploy the SD-CORE UPF charmed operator as a standalone network 
-function on Kubernetes.
+This guide explains how to deploy the SD-CORE User Plane charm bundle.
 
 ## Requirements
 
 - Juju >= 3.0
```

Comment:
We're using Juju `3.1.X` in the CI pipelines, should we require the same?

Reply:
Fixed